demisto_sdk/commands/generate_docs/tests/test_files/mirroring_test_markdow

```diff
@@ -0,0 +1,23 @@
+## Incident Mirroring
+
+You can enable incident mirroring between Cortex XSOAR incidents and CrowdStrike Falcon %%incident type%% (available from Cortex XSOAR version 6.0.0).
```

Comment:
why the `%%`?

Reply:
it should be filled by the developer according to the relevant incident type of the integration.